src/test/java/io/jenkins/plugins/analysis/core/charts/SeverityPieChartTest.java

```diff
@@ -0,0 +1,87 @@
+package io.jenkins.plugins.analysis.core.charts;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link SeverityPieChart}.
+ *
+ * @author Matthias Herpers
+ */
+class SeverityPieChartTest {
+
+    /**
+     *  Verifies that a SeverityPieChart with empty report shows only predefined severities without Severity.ERROR.
+     */
+    @Test
+    void shouldCreateChartFromEmptyReport() {
+
+        SeverityPieChart severityPieChart = new SeverityPieChart();
+        int numberOfDefaultPieData = Severity.getPredefinedValues().size() - 1; // Without Severity.ERROR
+        Report report = createReport(0, 0, 0, 0);
+        PieChartModel pieChartModel = severityPieChart.create(report);
+        List<PieData> data = pieChartModel.getData();
+        for (PieData severity : data) {
+            assertThat(Severity.getPredefinedValues().contains(severity));
```

Comment:
This does not check anything, use
```
assertThat(actual).isEqualTo(expected);
assertThat(actualBoolean).isTrue();
```
See https://joel-costigliola.github.io/assertj/assertj-core-quick-start.html.

See also https://ci.jenkins.io/job/Plugins/job/warnings-ng-plugin/job/PR-47/6/spotbugs/new/